test/fixed-buf-iter.c

```diff
@@ -87,6 +87,9 @@ static int test(struct io_uring *ring)
 		return 1;
 	}
 	io_uring_cqe_seen(ring, cqe);
+	for (i = 0; i < BUFFERS; i++) {
```

Comment:
Parens.

Reply:
I'm sorry, I don't understand what you mean. I copied this from line 39, which I didn't touch.

Reply:
Sorry it's braces, don't need {} for a single line statement.